test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Spatial/SpatialController.cs

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNet.OData;
+using Microsoft.Spatial;
+using Microsoft.Test.E2E.AspNet.OData.Common.Controllers;
+using Xunit;
+
+namespace Microsoft.Test.E2E.AspNet.OData.Spatial
+{
+    public class SpatialCustomersController : TestODataController
+    {
+        private static readonly IEnumerable<SpatialCustomer> Customers;
+
+        static SpatialCustomersController()
+        {
+            if (Customers == null)
+            {
+                string[] names = { "Mercury", "Venus", "Earth", "Mars", "Jupiter", "Saturn", "Uranus", "Neptune" };
+
+                Customers = Enumerable.Range(1, 7).Select(e => new SpatialCustomer
+                {
+                    CustomerId = e,
+                    Name = names[e - 1],
```

Comment:
Why do we take only a range of the string names?

Reply:
Just sample data.